3441.xml

```diff
@@ -0,0 +1,243 @@
+                <Type>Integer</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description> Initial value must be "1024".
+                </Description>
+            </Item>
+            <Item ID="30">
+                <Name>Float Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Float</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be "3.14159".
+                </Description>
+            </Item>
+            <Item ID="40">
+                <Name>Boolean Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Boolean</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be "true".</Description>
+            </Item>
+            <Item ID="50">
+                <Name>Opaque Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Opaque</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be "0x0123456789ABCDEF".
+                </Description>
+            </Item>
+            <Item ID="60">
+                <Name>Time Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Time</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be the time to an 1st,
+                    2000 in the UTC time zone. (Timestamp value :
+                    946684800)</Description>
+            </Item>
+            <Item ID="70">
+                <Name>ObjLink Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Objlnk</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be a link to instance 0
+                    of Device Object 3 (3:0).</Description>
+            </Item>
+            <Item ID="1010">
+                <Name>Multiple String Value</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Multiple</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>String</Type>
+                <RangeEnumeration />
+                <Units />
+                <Description>Initial value must be 1 instance with id 0
```

Comment:
```suggestion
                <Description>Initial value must be 1 instance with ID 0
```
(and same with other instances of "id" too)

Reply:
fixed by commit 8c7ee24